### DISCUSSION TITLE
mongodb support for updates using aggregation pipelines

### DISCUSSION THREAD

**Discussion by Loki-Afro** [mikro-orm/mikro-orm#6176] [Ideas]
**Is your feature request related to a problem? Please describe.**
While writing a migration i encountered that I'm unable to use this mongodb feature https://www.mongodb.com/docs/manual/tutorial/update-documents-with-aggregation-pipeline/ 

i think that feature existed already back in mongodb 4.2

**Describe the solution you'd like**
be able to use `this.driver.nativeUpdateMany...` with an aggregation pipeline or similar

**Comment by B4nan** [comment]
How would it be better than the current solution? This proposal is very vague. I am personally not interested in implementing this, but I would accept a PR.

Note that you can use whatever native feature you want via `em.getCollection(Foo)....`, followed by `em.map()` if you want to map the raw results to entity instances.

**Reply by Loki-Afro** [reply]
ah, `getCollection()` did the trick, thank you for the hint
